Animations/Add-animation-effect-to-PowerPoint-image-shape-text/.NET/Add-animation-effect-to-PowerPoint-image-shape-text/Add-animation-effect-to-PowerPoint-image-shape-text.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <RootNamespace>Add-animation-effect-to-PowerPoint-image-shape-text</RootNamespace>
```

Comment:
Refer and maitain same structure and csproj
https://github.com/SyncfusionExamples/DocIO-Examples/pull/352/files